dvc/scm.py

```diff
@@ -123,3 +125,56 @@ def resolve_rev(scm: "Git", rev: str) -> str:
             if len(ref_infos) > 1:
                 raise RevError(f"ambiguous Git revision '{rev}'")
         raise RevError(str(exc))
+
+
+def iter_revs(
+    scm: "Git",
+    head_revs: Optional[List[str]] = None,
```

Comment:
Why not use `revs`?

Reply:
We can call it `revs` + `results` compare to current `head_revs` + `revs`